Gemfile

```diff
@@ -8,7 +8,7 @@ gem 'ejson'
 gem 'fog'
 gem 'fog-json'
 gem 'fog-xml'
-gem 'fog-dynect'
+gem 'fog-dynect', :git => 'https://github.com/Shopify/fog-dynect.git'
```

Comment:
Is there a reason we didn't PR the change to upstream? I'd be very hesitant to start maintaining a fork.